plugins/modules/fabric_devices_workflow_manager.py

```diff
@@ -0,0 +1,4080 @@
+                        - local_autonomous_system_number can be from 1 to 4,294,967,295.
+                        - Dot notation is also allowed. Varies from 1.0 to 65535.65535. For example, 65534.65535
+                        type: str
+                      is_default_exit:
+                        description:
+                        - Default gateway used for routing traffic from the fabric to external networks.
+                        - Provides a path for traffic to exit the virtual network via the Border Node.
+                        - Default value is True.
+                        - is_default_exit can be updated.
+                        type: bool
+                      import_external_routes:
+                        description:
+                        - Prevents the import of routes learned from external networks into the fabric.
+                        - Ensures that only internal routes within the SDA fabric are used, enhances security.
+                        - Default value is True.
+                        - import_external_routes can be updated.
+                        type: bool
+                      border_priority:
+                        description:
+                        - Determines the preference level for a Border Node when multiple border nodes are available.
+                        - A higher-priority border node is preferred for routing traffic to external networks.
+                        - border_priority should be between 1 to 9.
+                        - If border_priority is not set, the default value is 10.
+                        - border_priority can be updated.
+                        type: int
+                      prepend_autonomous_system_count:
+                        description:
+                        - Artificially increase the AS path length when advertising routes via BGP.
+                        - Makes a route appear less favorable to external peers.
+                        - prepend_autonomous_system_count should be between 1 to 10.
+                        - If prepend_autonomous_system_count is not set, the default value is 0.
+                        - prepend_autonomous_system_count can be updated.
+                        type: int
+                  layer3_handoff_ip_transit:
+                    description:
+                    - Adds layer 3 handoffs with ip transit in fabric devices.
+                    - Configured when IP traffic is routed from the SDA fabric to external networks.
+                    - If layer3_handoff_ip_transit is set, border type will be considered as Layer3.
+                    type: list
+                    elements: dict
+                    suboptions:
+                      transit_network_name:
+                        description:
+                        - Network that connects multiple SDA fabrics or networks.
+                        - Mandatory parameter for all operations in L3 Handoff with IP transit.
```

Comment:
- Required for all operations in L3 Handoff with IP transit.
                    - It is not possible to update `transit_network_name` after initial configuration.